### ISSUE TITLE
Questions about the loss function of the Pytorch

### ISSUE THREAD

**Issue by liuyeqi94** [ialhashim/DenseDepth#221]
Thanks a lot for making such an awesome research. I found some problems in running the code and would like to ask you for advice.
In the Pytorch version of the code, the loss function in train.py consists of two parts, L1loss and SSIM, but the loss function in the paper consists of three parts, L1 loss, the L1 loss defined over the image gradient g of the depth image, and SSIM loss. Why is the second loss function missing from the loss function in the code? Is the impact significant? I am confused.
Thanks again for your contribution and I am looking forward to your reply.
